NSS-Day-Cohort-53/kennels-voyagers
# Src search

```text
#### Changes Made
1. Modified file `SearchResults.js` to include `search results showing` and functionality. Also made it so that customers can no longer search for animals
2. Modified file `SearchResults.css` to include `styling` 
​
#### Steps to Review
1. Checkout this branch locally.
    ```
    git fetch --all
    git checkout src-search
    ```
2. Open a new Terminal tab (⌘T) and navigate to the server directory.
3. Test app functionality.
    > start the app
    > log in as an employee
    > search something simple like "a" and see if all results show up
    > click on a link from each of the three categories to see that they pull up the individual cards

    > log in as an owner
    > search something simple like "a" again and make sure animals don't show up
4. View code file.
    > Confirm file modifications are present as indicated above.
    > Confirm no unused code or extraneous comments exist.
```
